source/chapters/appendix.rst

```diff
@@ -77,9 +71,9 @@ language settings the defaults may be different.
 +----------------------------------------+-----------------------------------------+------------------------------------------+----------------------------+
 | **Playback**                                                                                                                                             |
 +----------------------------------------+-----------------------------------------+------------------------------------------+----------------------------+
-| Load selected track to deck            | :kbd:`Shift` + :kbd:`left`              | :kbd:`Shift` + :kbd:`right`              |                            |
+| Load selected track to deck            | :kbd:`Shift` + :kbd:`←`                 | :kbd:`Shift` + :kbd:`→`                  |                            |
 +----------------------------------------+-----------------------------------------+------------------------------------------+----------------------------+
-| Unload track from deck                 | :kbd:`Alt` + :kbd:`Shift` + :kbd:`left` | :kbd:`Alt` + :kbd:`Shift` + :kbd:`right` |                            |
+| Unload track from deck                 | :kbd:`Alt` + :kbd:`Shift` + :kbd:`←`    | :kbd:`Alt` + :kbd:`Shift` + :kbd:`→`     |                            |
```

Comment:
Here as well:
Please revert that since we agreed to keep the literal key names because there are obviously no icons that work reliably on all platforms.

Reply:
I thought we just said we use the regular arrows instead of the more obscure ones.